docs/_data/authors.yml

```diff
@@ -130,6 +130,14 @@ veevidify:
   web:
   bio:
 
+
+ray6464:
+  name: Ray Voice
+  email: ray@vanet@gmail.com
+  web: twitter: @rayshorthead
```

Comment:
Looks like this syntax breaks the YAML. Could you link the Twitter URL here?

Reply:
fixed